specifications/DiningPhilosophers/DiningPhilosophers.tla

```diff
@@ -113,7 +113,7 @@ Think:
 end process;
 
 end algorithm; *)
-\* BEGIN TRANSLATION (chksum(pcal) = "8d8268d4" /\ chksum(tla) = "16352822")
+\* BEGIN TRANSLATION (chksum(pcal) = "ea877089" /\ chksum(tla) = "16352822")
```

Comment:
FYI, the Toolbox raises a warning if the PlusCal or the translated TLA+ do not match their checksums, respectively. However, a user may write `chksum(pcal) \in STRING` or `chksum(tla) \in STRING` to indicate that the pcal or TLA+ has been edited after translation. If the goal of this PR is to always retranslate, I'd suggest honoring the same convention and skipping retranslation.

Reply:
Hmmm okay, so this would be as simple as searching for the string `chksum(pcal) \in STRING` in the file then skipping the translation if so?

Reply:
The logic is implemented in the Pcal translator: https://github.com/tlaplus/tlaplus/blob/master/tlatools/org.lamport.tlatools/src/pcal/Validator.java

Reply:
Okay so it's fine to re-run translation on every file? Also thinking about emitting an error in the CI if running the translator results in a mismatch.

Reply:
Sounds fine to me to report an error if there is a checksum mismatch.